.github/workflows/gce_test.yaml

```diff
@@ -0,0 +1,103 @@
+name: GlobusComputeExecutor tests
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+jobs:
+  main-test-suite:
+    strategy:
+      matrix:
+        python-version: ["3.11"]
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+
+    steps:
+    - uses: actions/checkout@master
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Collect Job Information
+      id: job-info
+      run: |
+        echo "Python Version: ${{ matrix.python-version }}" >> ci_job_info.txt
+        echo "CI Triggering Event: ${{ github.event_name }}" >> ci_job_info.txt
+        echo "Triggering Git Ref: ${{ github.ref }}" >> ci_job_info.txt
+        echo "Triggering Git SHA: ${{ github.sha }}" >> ci_job_info.txt
+        echo "Workflow Run: ${{ github.run_number }}" >> ci_job_info.txt
+        echo "Workflow Attempt: ${{ github.run_attempt }}" >> ci_job_info.txt
+        as_ascii="$(echo "${{ github.ref_name }}" | perl -pe "s/[^A-z0-9-]+/-/g; s/^-+|-+\$//g; s/--+/-/g;")"
+        echo "as-ascii=$as_ascii" >> $GITHUB_OUTPUT
+
+    - name: Non-requirements based install
+      run: |
+        # libpython3.5: make workqueue binary installer happy
+        # mpich: required by radical executor
+        sudo apt-get update -q
+        sudo apt-get install -qy libpython3.5 mpich
+
+    - name: setup virtual env
+      run: |
+        make virtualenv
+        source .venv/bin/activate
+
+    - name: make deps clean_coverage
+      run: |
+        source .venv/bin/activate
+        make deps
+        make clean_coverage
+        
+        # Temporary fix, until changes make it into compute releases
+        git clone -b configure_tasks_working_dir https://github.com/globus/globus-compute.git        
```

Comment:
maybe its good to wait for Globus Compute to support the necessary features in a release, so that this can pin a relevant supported release?

Reply:
(does this mean that right now a user shouldn't expect to be able to use an actual Globus Compute but instead needs to install this fork?)

Reply:
The github action uses `ThreadPoolEngine` because it is lightweight compared to running the default config that uses `GlobusComputeEngine`. The branch that I'm pointing to fixes a `ThreadPoolEngine` only bug. You can run these same tests against a local GlobusCompute endpoint, by setting `GLOBUS_COMPUTE_ENDPOINT=<ep_id>` and I've done those tests against a few configs. I believe Chris (NOAA) has also done a bunch of testing at this point.

Reply:
I have been testing the `GlobusComputeExecutor` via this branch in combination with `globus-compute-endpoint==2.30.1`.  I am using that release, not a special branch/tag.  The one feature I haven't yet tested is passing in the `user_endpoint_config` at call time rather than at `GlobusComputeExecutor` instantiation time.  What I've done so far is working great!